packages/module/src/utils/css.tsx

```diff
@@ -0,0 +1,27 @@
+export function css(...args: any): string {
```

Comment:
Just copy+pasted this from the react repo to use to clean up some whitespace in the class attribute in the DOM